src/documentation/pages/components/forms/range-slider/examples/_radio_consistent.vue

```diff
@@ -0,0 +1,90 @@
+<template lang="pug">
+  <ComponentExample title="Consistent rendering in WebKit browsers" id="radio-consistent" :tabs="exampleTabs">
+    .chi-form__item(slot="example")
+      label(class="chi-label", id="range-label", for="range09") Range label
+      .chi-input__wrapper
+        input(class="chi-range-slider", type="range", min="0", max="4", step="1", ref="range")
```

Comment:
On label you are using `for="range09"`
```suggestion
        input(class="chi-range-slider", type="range", min="0", max="4", step="1", ref="range" id="range09")
```